schedule/functional/leap16/agama_create_hdd_textmode_leap.yaml

```diff
@@ -0,0 +1,19 @@
+name:  agama_create_hdd_textmode_leap
+description:    >
+    Install sle16 via agama auto and publish qcow2 images
+conditional_schedule:
+    grub_test:
+        ARCH:
+            'aarch64':
+                - installation/grub_test
+schedule:
+    - yam/agama/boot_agama
+    - yam/agama/agama_auto
+    - '{{grub_test}}'
```

Comment:
If possible, we can remove the test module `grub_test`

Reply:
For aarch64, after reboot, we need choose boot from hard disk like: https://openqa.opensuse.org/tests/4774712#step/grub_test/5